holoviews/util/transform.py

```diff
@@ -106,6 +106,36 @@ def categorize(values, categories, default=None):
     return np.asarray(cats)
 
 
+def _maybe_map(numpy_fn):
```

Comment:
Was wondering whether we should do this for any numpy function that's not implemented but I think for now sticking with just isin and digitize is safest.